pkg/oc/cli/cmd/login/loginoptions.go

```diff
@@ -226,8 +226,13 @@ func (o *LoginOptions) gatherAuthInfo() error {
 	clientConfig.KeyFile = o.KeyFile
 	token, err := tokencmd.RequestToken(o.Config, o.Reader, o.Username, o.Password)
 	if err != nil {
+		if kerrors.IsUnauthorized(err) {
```

Comment:
I think its time to gut this from here and move this logic to `pkg/oc/cli/cmd/login/login.go` where we handle the `IsUnauthorized` error.

Reply:
👍 @juanvallejo would you mind?

Reply:
Thanks, moved to cmd/login/login.go